index.bs

```diff
@@ -5820,30 +5820,51 @@ Note: In order to interoperate, user agents storing large blobs on authenticator
 :: `largeBlob`
 
 : Operation applicability
-:: [=authentication extension|Authentication=]
+:: [=registration extension|Registration=] and [=authentication extension|authentication=]
 
 : Client extension input
 ::  <xmp class="idl">
     partial dictionary AuthenticationExtensionsClientInputs {
         AuthenticationExtensionsLargeBlobInputs largeBlob;
     };
 
+    enum LargeBlobSupport {
+      "required",
+      "preferred",
+    };
+
     dictionary AuthenticationExtensionsLargeBlobInputs {
+        DOMString support;
         boolean read;
         ArrayBuffer write;
     };
     </xmp>
 
     <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsLargeBlobInputs">
+        :   <dfn>support</dfn>
+        ::  A DOMString that takes one of the values of {{LargeBlobSupport}}. (See [[#sct-domstring-backwards-compatibility]].) Only valid during [=registration extension|registration=].
+
         :   <dfn>read</dfn>
-        ::  A boolean that indicates that the [=[RP]=] would like to fetch the previously-written blob associated with the asserted credential.
+        ::  A boolean that indicates that the [=[RP]=] would like to fetch the previously-written blob associated with the asserted credential. Only valid during [=authentication extension|authentication=].
 
         :   <dfn>write</dfn>
-        ::  An opaque byte string that the [=[RP]=] wishes to store with the existing credential.
+        ::  An opaque byte string that the [=[RP]=] wishes to store with the existing credential. Only valid during [=authentication extension|authentication=].
     </div>
 
+: Client extension processing ([=registration extension|registration=])
+::
+       1. If {{AuthenticationExtensionsLargeBlobInputs/read}} or {{AuthenticationExtensionsLargeBlobInputs/write}} is present:
+           1. Return a {{DOMException}} whose name is “{{NotSupportedError}}”.
+       1. If {{AuthenticationExtensionsLargeBlobInputs/support}} is present and has the value “required”:
```

Comment:
It seems to me that there is no point in using the registration extension with a `AuthenticationExtensionsLargeBlobInputs/support` that is not present. Perhaps we should default to `preferred` or throw an error.

Reply:
Done. (Defaulting to “preferred”.)

Reply:
I do not see where we "default to 'preferred'" ?